docs/configure-tls.md

```diff
@@ -0,0 +1,591 @@
+<!--[metadata]>
++++
+title = "Configure Docker Swarm for TLS"
+description = "Swarm and transport layer security"
+keywords = ["docker, swarm, TLS, discovery, security,  certificates"]
+[menu.main]
+parent="workw_swarm"
+weight=55
++++
+<![end-metadata]-->
+
+# Configure Docker Swarm for TLS
+
+In this procedure you create a two-node Swarm cluster, a Docker Engine CLI, a
+Swarm Manager, and a Certificate Authority as shown below. All the Docker Engine
+hosts (`client`, `swarm`, `node1`, and `node2`) have a copy of the
+CA's certificate as well as their own key-pair signed by the CA.
+
+![](images/tls-1.jpg)
+
+You will complete the following steps in this procedure:
+
+
+- [Step 1: Set up the prerequisites](#step-1-set-up-the-prerequisites)
+- [Step 2: Create a Certificate Authority (CA) server](#step-2-create-a-certificate-authority-ca-server)
+- [Step 3: Create and sign keys](#step-3-create-and-sign-keys)
+- [Step 4: Install the keys](#step-4-install-the-keys)
+- [Step 5: Configure the Engine daemon for TLS](#step-5-configure-the-engine-daemon-for-tls)
+- [Step 6: Create a Swarm cluster](#step-6-create-a-swarm-cluster)
+- [Step 7: Create the Swarm Manager using TLS](#step-7-create-the-swarm-manager-using-tls)
+- [Step 8: Test the Swarm manager configuration](#step-8-test-the-swarm-manager-configuration)
+- [Step 9: Configure the Engine CLI to use TLS](#step-9-configure-the-engine-cli-to-use-tls)
+
+### Before you begin
+The article includes steps to create your own CA using OpenSSL. This is similar
+to operating your own internal corporate CA and PKI. However, this `must not`
+be used as a guide to building a production-worthy internal CA and PKI. These
+steps are included for demonstration purposes only - so that readers without
+access to an existing CA and set of certificates can follow along and configure
+Docker Swarm to use TLS.
+
+
+## Step 1: Set up the prerequisites
+
+To complete this procedure you must stand up 5 (five) Linux servers. These
+servers can be any mix of physical and virtual servers; they may be on premises
+or in the public cloud. The following table lists each server name and its purpose.
+
+| Server name | Description                                    |
+|-------------|------------------------------------------------|
+| `ca`      | Acts as the Certificate Authority (CA) server. |
+| `swarm`   | Acts as the Swarm Manager.                     |
+| `node1`   | Act as a Swarm node.                           |
+| `node2`   | Act as a Swarm node.                           |
+| `client`  | Acts as a remote Docker Engine client          |
+
+Make sure that you have SSH access to all 5 servers and that they can communicate with each other using DNS name resolution. In particular:
+
+- TCP port 2376 must be open between the Swarm Manager and Swarm nodes
+- TCP port 3376 must be open between the Docker Engine client and the Swarm Manager

```

Comment:
is using port `3376` a common choice for putting the swarm server on?

These seems fraught with all the same problems we had before we registered `2375` and `2376` as official Docker TCP ports.

if it _is_ intentional, but not common to all the swarm docs, please mention that 'you should pick your own port'